changes/1234.doc.rst

```diff
@@ -0,0 +1,2 @@
+Changed the machinery for creating changelog entries.
```

Comment:
Shouldn't this file be named be #2736 or #829?

Reply:
Woops, thanks for catching